sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample7_ConsumeEvents.cs

```diff
@@ -103,7 +103,7 @@ await using (EventHubProducer producer = client.CreateProducer(new EventHubProdu
                     // time that we've allowed.
                     //
                     // We will ask for just our event, but allow a fairly long wait period to ensure that we're able to receive it.
-                    // If you observe the time that the call takes, it is extremely likely that the request to recieve will complete
+                    // If you observe the time that the call takes, it is extremely likely that the request to receive will complete
```

Comment:
Thank you for continuing to correct these; this seems to be a favorite error of mine.